roles/confluent.test/tasks/file_exists.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Check directory exists
+  shell: "find {{custom_path}}{{file_name}}"
+  register: file_exists
+    
+- name: Assert log file exists
```

Comment:
could you combine the file_exists and check directory task file, seems like they both could do the same thing (check the file exists and ownership)